services/api/service.go

```diff
@@ -71,6 +71,9 @@ var (
 	apiReadHeaderTimeoutMs = cli.GetEnvInt("API_TIMEOUT_READHEADER_MS", 600)
 	apiWriteTimeoutMs      = cli.GetEnvInt("API_TIMEOUT_WRITE_MS", 10000)
 	apiIdleTimeoutMs       = cli.GetEnvInt("API_TIMEOUT_IDLE_MS", 3000)
+
+	// number of goroutines to pull from the optimistic block channel.
+	numOptimisticBlockProcessors = cli.GetEnvInt("NUM_OPTIMISTIC_BLOCK_PROCESSORS", 10)
```

Comment:
Consider increasing 10 to 100 or 1,000 if the overhead of the single goroutines is small.

Reply:
set to 100 for now! we can increase it if we find ourselves needing more.